### ISSUE TITLE
Serverless support coming?

### ISSUE THREAD

**Issue by onionhammer** [Azure/azure-cosmosdb-bulkexecutor-dotnet-getting-started#72]
Would be good to know if this library will eventually support severless; or rather will Serverless ever support whatever APIs this library is using

**Comment by ThomasWeiss** [comment]
@onionhammer yes this gap is identified and we intend to address it before the serverless offer goes GA.

**Comment by onionhammer** [comment]
@ThomasWeiss is there some vague ETA on when serverless is going GA or when this support will be added? like Q2 2021 or something?
